core/drivers/pmd.h

```diff
@@ -143,8 +146,11 @@ class PMDPort final : public Port {
   }
 
   uint64_t NICCycleToCPUCycle(u_int64_t nic_cycle) {
-    return linear_re_.GetY(nic_cycle);
-  }
+    linear_re_lock_.lock_shared();
```

Comment:
Add a comment saying that the current implementation incurs a per-packet overhead, which can be further optimized.